src/api/activity/notifications.rs

```diff
@@ -0,0 +1,351 @@
+//! Github Notifications API
+
+use crate::models::activity::Notification;
+use crate::models::activity::ThreadSubscription;
+use crate::Octocrab;
+use crate::Page;
+
+type DateTime = chrono::DateTime<chrono::Utc>;
+
+/// Handler for GitHub's notifications API.
+///
+/// Created with [`ActivityHandler::notifications`].
+/// **Note:** All of these methods require authentication using
+/// your GitHub Access Token with the right privileges.
+///
+/// [`ActivityHandler::notifications`]: ../struct.ActivityHandler.html#method.notifications
+pub struct NotificationsHandler<'octo> {
+    crab: &'octo Octocrab,
+}
+
+impl<'octo> NotificationsHandler<'octo> {
+    pub(crate) fn new(crab: &'octo Octocrab) -> Self {
+        Self { crab }
+    }
+
+    /// Gets a notification by their id.
+    ///
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    /// let crab = octocrab::Octocrab::builder()
```

Comment:
I think I would prefer to create the `Octocrab` instance in the examples using `octocrab::instance()`. It's less boilerplate.